torch/distributed/nn/api/remote_module.py

```diff
@@ -50,6 +57,16 @@ def __init__(
         The arguments of ``forward_async`` and ``forward`` are the same as
         the ``forward`` method of the module returned by the ``module_cls``.
 
+        Since all the other methods except ``forward`` and ``forward_async``
+        directly inherit from ``nn.Module``,
+        these methods are only applied to the caller on the local side,
```

Comment:
Maybe reword this a bit, without reading the following sentence it seems like these methods actually work for RemoteModule and are applied on the local side. Maybe we can just mention that apart from `forward` and `forward_async` no other methods are supported from nn.Module.